cmd/earthly/main.go

```diff
@@ -617,17 +617,20 @@ func (app *earthlyApp) run(ctx context.Context, args []string) int {
 			}
 		}
 
-		if strings.Contains(err.Error(), "security.insecure is not allowed") {
+		switch {
+		case strings.Contains(err.Error(), "security.insecure is not allowed"):
 			app.logbus.Run().SetFatalError(time.Now(), "", "", logstream.FailureType_FAILURE_TYPE_NEEDS_PRIVILEGED, err.Error())
 			app.console.Warnf("Error: earthly --allow-privileged (earthly -P) flag is required\n")
-		} else if strings.Contains(err.Error(), "failed to fetch remote") {
+			return 9
```

Comment:
Out of curiosity, why is this one `9` and most of the rest `1`?
Should we return a unique number for each case?

Reply:
Are these exit codes? I agree that it would be nice to use named consts or add comments. Minor thing.

Reply:
They are - but it's pretty self-explanatory what they indicate given the warnings printed for them. Nothing really depends on these specific exit codes currently. I figured I would give special number numbers to some specific, more important, situations as a nice-to-have.